src/procedures/TransferSecurityTokens.ts

```diff
@@ -55,6 +61,11 @@ export class TransferSecurityTokens extends Procedure<TransferSecurityTokensProc
     }
   }
 
+  /**
+   * - Send tokens to wallet address
```

Comment:
```suggestion
   * Transfer security tokens from a wallet address to another
```
It should also mention that if `from` isn't provided, it will be the current sdk user address by default